bowtie/templates/report.html.j2

```diff
@@ -278,8 +282,7 @@
               {% for test, test_results in results %}
               <tr>
                 <td>
-                  <a href="#" data-bs-toggle="tooltip" data-bs-html="true"
-                    data-bs-title='<pre>{{ test.instance | tojson(indent=2) }}</pre>'>{{ test.description }}</a>
+                  <a href="#schema" class="text-decoration-none" onclick='displayCode(`{{ test.instance | tojson(indent=2) }}`,"instance-info")'>{{ test.description }}</a>
```

Comment:
I think this `onClick` should go higher -- right now it only renders the instance if you click on the text, but someone is likely to click on the *row*, even beyond the text. So maybe on the `tr`? Until/unless we have some way of displaying the actual results more clearly.